File: data/java/Function.java
package nameOfThePackage;

import com.amazonaws.services.lambda.runtime.Context;

public class Function {

    public static class FunctionRequest {
        String hello;

        public String getHello() {
            return hello;
        }

        public void setHello(String hello) {
            this.hello = hello;
        }

        public FunctionRequest(String hello) {
            this.hello = hello;
        }

        public FunctionRequest() {
        }
    }

    public static class FunctionResponse {
        String hello;

        public String getHello() {
            return hello;
        }

        public void setHello(String hello) {
            this.hello = hello;
        }

        public FunctionResponse(String hello) {
            this.hello = hello;
        }

        public FunctionResponse() {
        }
    }

    public FunctionResponse handler(FunctionRequest event, Context context) {
        return new FunctionResponse(event.getHello());
    }
}

File: data/kotlin/Function.kt
package nameOfThePackage

import com.amazonaws.services.lambda.runtime.Context

class Function {

    class ExampleRequest {
        var hello: String = "azaza"

        constructor(hello: String) {
            this.hello = hello
        }

        constructor() {}
    }

    class ExampleResponse {
        var hello: String = "zazazaza"

        constructor(hello: String) {
            this.hello = hello
        }

        constructor() {}
    }

    fun handler(event: ExampleRequest, context: Context): ExampleResponse {
        return ExampleResponse(event.hello)
    }
}
